[2025-05-06T16:55:55.382+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: hello_airflow.print_hello scheduled__2025-03-21T00:00:00+00:00 [queued]>
[2025-05-06T16:55:55.396+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: hello_airflow.print_hello scheduled__2025-03-21T00:00:00+00:00 [queued]>
[2025-05-06T16:55:55.397+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-05-06T16:55:55.580+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): print_hello> on 2025-03-21 00:00:00+00:00
[2025-05-06T16:55:55.588+0000] {standard_task_runner.py:60} INFO - Started process 1085 to run task
[2025-05-06T16:55:55.592+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'hello_***', 'print_hello', 'scheduled__2025-03-21T00:00:00+00:00', '--job-id', '564', '--raw', '--subdir', 'DAGS_FOLDER/example.py', '--cfg-path', '/tmp/tmpi3dh5zj1']
[2025-05-06T16:55:55.595+0000] {standard_task_runner.py:88} INFO - Job 564: Subtask print_hello
[2025-05-06T16:55:55.616+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.8/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2025-05-06T16:55:55.666+0000] {task_command.py:423} INFO - Running <TaskInstance: hello_airflow.print_hello scheduled__2025-03-21T00:00:00+00:00 [running]> on host afa31441e313
[2025-05-06T16:55:55.794+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='hello_***' AIRFLOW_CTX_TASK_ID='print_hello' AIRFLOW_CTX_EXECUTION_DATE='2025-03-21T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-03-21T00:00:00+00:00'
[2025-05-06T16:55:55.796+0000] {logging_mixin.py:188} INFO - Hello, Airflow! DAG đang chạy thành công.
[2025-05-06T16:55:55.797+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-05-06T16:55:55.810+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=hello_***, task_id=print_hello, execution_date=20250321T000000, start_date=20250506T165555, end_date=20250506T165555
[2025-05-06T16:55:55.846+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-05-06T16:55:55.880+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
